src/github.com/couchbase/sync_gateway/db/change_cache.go

```diff
@@ -186,12 +186,15 @@ func (c *changeCache) CleanSkippedSequenceQueue() bool {
 		for _, skippedSeq := range c.skippedSeqs {
 			if time.Since(skippedSeq.timeAdded) > c.options.CacheSkippedSeqMaxWait {
 				// Attempt to retrieve the sequence from the view before we remove
-				options := ChangesOptions{Since: SequenceID{Seq: skippedSeq.seq}}
+				options := ChangesOptions{Since: SequenceID{Seq: skippedSeq.seq - 1}}
```

Comment:
Can you factor out `skippedSeq.seq - 1` into a named variable?  Also if it's not possible to say what it represents completely with the name, can you add some comments that explain what it is?